docs/src/vis.md

```diff
@@ -0,0 +1,53 @@
+# Visualization
+
+SparseMatrixColorings provides some internal utilities for visualization of matrix colorings via the un-exported function `show_colors`.
+
+!!! warning
+    Using this function requires loading [Images.jl](https://github.com/JuliaImages/Images.jl).
+
+## Visualizing results
+
+Currently, only `ColumnColoringResult` and `RowColoringResult` are supported.
+
+```@example img
+using Images
```

Comment:
```suggestion
using ColorTypes
```

Reply:
Images.jl is still required to show images within the doc page.